src/vstart.sh

```diff
@@ -366,13 +366,24 @@ fi
 
 ARGS="-c $conf_fn"
 
+quoted_print() {
+    for s in "$@"; do
+        if [[ "$s" =~ \  ]]; then
+            printf -- "'%s' " "$s"
+        else
+            printf -- "$s "
+        fi
+    done
+    printf '\n'
+}
+
 prunb() {
-    printf "'%s' " "$@"; echo '&'
+    quoted_print $* '&'
```

Comment:
This should be `quoted_print "$@" '&'`?

Reply:
@batrick thanks for pointing this out, fixed at #18523